packages/react-core/src/Authenticator/hooks/types.ts

```diff
@@ -65,18 +65,18 @@ type RouteComponent<
  * Common component prop types used for both RWA and RNA implementations
  */
 interface CommonRouteProps extends ComponentSlots {
-  error: AuthenticatorMachineContext['error'];
+  error?: AuthenticatorMachineContext['error'];
   fields: AuthenticatorLegacyFields;
   isPending: AuthenticatorMachineContext['isPending'];
 }
 
 export interface CommonConfirmResetPasswordProps extends CommonRouteProps {
   resendCode: AuthenticatorMachineContext['resendCode'];
-  validationErrors: AuthenticatorMachineContext['validationErrors'];
+  validationErrors?: AuthenticatorMachineContext['validationErrors'];
 }
 
 export interface CommonConfirmSignInProps extends CommonRouteProps {
-  challengeName: AuthChallengeName;
+  challengeName?: AuthChallengeName;
```

Comment:
Should this be required? We do `challengeName` check before state machine transitions to `confirmSignIn`: https://github.com/aws-amplify/amplify-ui/blob/5a0a836314e606cb58b340272b1c45b3dc608050/packages/ui/src/machines/authenticator/actors/signIn.ts#L485-L487

so `challengeName` should always be present if I understand correctly.

Reply:
```suggestion
  challengeName: AuthChallengeName;
```
Was initially thinking it about it in terms of users passing override components, and if they want to leave it unused for some reason. Since it's always provided think it makes sense ti add it back